devel/tilt/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/tilt-dev/tilt 0.18.1 v
+revision            0
+
+categories          devel
+maintainers         nomaintainer
+platforms           darwin
+supported_archs     x86_64
```

Comment:
Unless you're sure i386 isn't supported, I'd use the default set by the golang portgroup.

https://github.com/macports/macports-ports/blob/1a1667c302671c5f1043338ac0bbd8e44acd3df2/_resources/port1.0/group/golang-1.0.tcl#L129